modules/nf-core/nacho/normalise/main.nf

```diff
@@ -0,0 +1,58 @@
+nextflow.enable.moduleBinaries = true
+
+process NACHO_NORMALISE {
+    tag "$sample_sheet"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container 'community.wave.seqera.io/library/r-dplyr_r-fs_r-ggplot2_r-nacho_pruned:033bc017f5f36b6d'
+
+    input:
+    path rcc_files, stageAs: "input/*"
+    path sample_sheet
+
+    output:
+    path "*normalized_counts.tsv"          , emit: normalized_counts
+    path "*normalized_counts_wo_HKnorm.tsv", emit: normalized_counts_wo_HK
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    nacho_norm.R --input_rcc_path input --input_samplesheet ${sample_sheet} $args
```

Comment:
```suggestion
    nacho_norm.R \\
      --input_rcc_path input \\
      $args \\
      --input_samplesheet ${sample_sheet} 
```

Reply:
Did you write `$args` in the middle line for any particular reason?